File: back end/src/main/java/com/spring/restuarant/model/Order.java
package com.spring.restuarant.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.Type;

import javax.persistence.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "orders")
public class Order extends CategoryOrder {

    private String orderName;
    private int price;
    private String img;
    @Lob
    @Type(type = "org.hibernate.type.TextType")
    private String description;

    @JsonIgnore
    @ManyToOne
    @JoinColumn(name = "category_id")
    private Category category;
}

File: front-end/src/model/order.ts
export class Order {

  id:number;
  order_name: string;
  data_created:Date;
  data_update:Date;
  description:string;
  image:string;
  price:number;


  constructor(id: number, order_name: string, data_created: Date, data_update: Date, description: string, image: string, price: number) {
    this.id = id;
    this.order_name = order_name;
    this.data_created = data_created;
    this.data_update = data_update;
    this.description = description;
    this.image = image;
    this.price = price;
  }

}
